product_equivalent_category/models/product_equivalent_category.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2019 ForgeFlow S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class ProductEquivalentCategory(models.Model):
+    _name = "product.equivalent.category"
+    _description = "Product Equivalent Category"
+
+    name = fields.Char("Name", index=True, required=True, translate=True)
+    product_count = fields.Integer(
+        "# Products",
+        compute="_compute_product_count",
+        help="The number of products under this category",
+    )
+
+    @api.multi
+    def _compute_product_count(self):
+        for categ in self:
+            categ.product_count = len(
+                self.env["product.template"].search(
```

Comment:
Is there a reason for not using search_count ?

Reply:
No reason at all, changed it. Thanks!